Sources/FeatherOpenAPIKitMacro/FeatherOpenAPIKitMacro.swift

```diff
@@ -0,0 +1,113 @@
+import SwiftCompilerPlugin
+import SwiftSyntax
+import SwiftSyntaxBuilder
+import SwiftSyntaxMacros
+import Foundation
+
+enum CustomError: Error { case message(String) }
```

Comment:
This could be a struct.

Reply:
I did not modify because of compiler user messages